src/handle-data.ts

```diff
@@ -52,8 +52,7 @@ export function getTotalPosition(layout: TableLayout) {
  */
 export function getTotalInfo(layout: TableLayout, colIdx: number, pageColIdx: number, numDims: number) {
   if (colIdx >= numDims) return layout.qHyperCube.qGrandTotalRow[colIdx - numDims]?.qText ?? '';
-  if (pageColIdx === 0) return layout.totals.label;
-  return '';
+  return pageColIdx === 0 ? layout.totals.label ?? '' : '';
```

Comment:
I would say that the old syntax is easier to ready, so something like this `if (pageColIdx === 0) return layout.totals.label ?? '';`